### DISCUSSION TITLE
Can someone help, netstat doesnt work on termux app

### DISCUSSION THREAD

**Discussion by Gitonomics** [termux/termux-packages#14063] [Q&A] [answered]
Can someone help, netstat -anp or netstat -punta doesnt work on termux mobile app, need to enable it is already installed, it says no support for AF INET (tcp) on this system or does give output with just header lines

**Comment by sylirre** [comment, marked as answer]
`netstat` as well as bunch of other networking information tools doesn't work on Android 10 and higher. This is not bug but is a result of new OS security design and obviously Termux can't change that.

**Reply by Gitonomics** [reply]
Then this is actually a loop hole of Android 10 and higher biggest threat to smartphone security then, hackers can get valueable information from the Mobile devices and misuse them, recently this is how credit card information of million android users was sold on Dark web as there was no way to retrieve who did what, is there anyother tool apart from termux which I could use on Android 10 or even higher version to detect suspicious connections to my devices.

**Reply by xtkoba** [reply]
I would suggest using an app which sets up an internal VPN server. (Personally I use "NoRoot Firewall", but I'm not sure if it works on Android 10+.)
